create_installer_state.py

```diff
@@ -77,6 +77,8 @@ def read_vesselinfo_from_file(filename):
       
     ValueError: Raised by rsa_string_to_publickey if the publickey string
       is improperly formated (IE not like "123 1234567").
+
+    IOError: Raised by open() if the vesselinfo file is missing.
```

Comment:
The docstring in main() (in this module) should also be updated to indicate that `IOError` is raised.  At the moment, it states: 
```IOError: if vessel.restrictions is not found, this must be included ahead of time in the target directory.```